ignite/src/main/java/org/hibernate/ogm/datastore/ignite/IgniteProperties.java

```diff
@@ -29,6 +32,36 @@
 	 */
 	public static final String IGNITE_INSTANCE_NAME = "hibernate.ogm.ignite.instance_name";
 
+	public static final String IGNITE_ALLOWS_TRANSACTION_EMULATION = "hibernate.ogm.ignite.allows_transaction_emulation";
+	/**
+	 * Configuration property 'concurrency' for {@link org.apache.ignite.IgniteTransactions#txStart(TransactionConcurrency, TransactionIsolation, long, int)}
+	 * @see org.apache.ignite.transactions.TransactionConcurrency
+	 */
+	public static final String IGNITE_TRANSACTION_CONCURRENCY = "hibernate.ogm.ignite.transaction.concurrency";
+	/**
+	 * Configuration property 'isolation' for {@link org.apache.ignite.IgniteTransactions#txStart(TransactionConcurrency, TransactionIsolation, long, int)}
+	 * @see org.apache.ignite.transactions.TransactionIsolation
+	 */
+	public static final String IGNITE_TRANSACTION_ISOLATION = "hibernate.ogm.ignite.transaction.isolation";
+	/**
+	 * Configuration property 'timeout' for {@link org.apache.ignite.IgniteTransactions#txStart(TransactionConcurrency, TransactionIsolation, long, int)}
+	 */
+	public static final String IGNITE_TRANSACTION_TIMEOUT = "hibernate.ogm.ignite.transaction.timeout";
+	/**
+	 * Configuration property 'txsize' for {@link org.apache.ignite.IgniteTransactions#txStart(TransactionConcurrency, TransactionIsolation, long, int)}
+	 */
+	public static final String IGNITE_TRANSACTION_TXSIZE = "hibernate.ogm.ignite.transaction.txsize";
+	/**
+	 * Configuration property 'support_persistence' activate Ignite Native Persistence feature
+	 * @see <a href="https://apacheignite.readme.io/docs/distributed-persistent-store">Ignite Native Persistence</a>
+	 * */
+	public static final String IGNITE_SUPPORT_PERSISTENCE = "hibernate.ogm.ignite.support_persistence";
+	/**
+	 * Configuration property 'work.directory.path' set path for work directory
+	 * @see <a href="https://apacheignite.readme.io/docs/distributed-persistent-store">Ignite Native Persistence</a>
+	 */
+	public static final String IGNITE_WORK_DIRECTORY = "hibernate.ogm.ignite.work.directory.path";
```

Comment:
Why did you do IGNITE_SUPPORT_PERSISTENCE and IGNITE_WORK_DIRECTORY properties?
They can be set trought CONFIGURATION_CLASS_NAME. There are many other properties in IgniteConfiguration. I don't think that work directory is something special

Reply:
For activate Ignite Native Persistence, Hibenate needs call "ignite.active(true);". You right that these properties can be set by configuration file/class. Any way, You need to send command to Hibernate to call "ignite.active(true);" after Ignite started.
In this way, property IGNITE_WORK_DIRECTORY can be removed from the class, but IGNITE_SUPPORT_PERSISTENCE is not.
Correct me please if I make mistake.

Reply:
IMHO trying to start real cluster, especially with local storage from Hibernate configuration is a big mistake. Better ways are to use OGM in Ignite client mode or to start cluster before Hibernate and then use IGNITE_INSTANCE_NAME to work with Ignite cluster. 
But if you still want to call active(true) you can do it always when clientMode=false or if persistentStoreConfiguration != null and clientMode=false.

Reply:
okey. Good idea!